.github/workflows/ci.yml

```diff
@@ -162,3 +162,48 @@ jobs:
           pip install tox
           tox -c examples/linalg
           tox -c examples/parallel
+
+  coverage:
+    name: coverage-${{ matrix.platform.os }}-${{ matrix.platform.arch }}-py${{ matrix.python }}
+    strategy:
```

Comment:
The single entry matrix looks a bit redundant? I think start with single platform coverage and avoid all the `${{ matrix.* }}` would make for an easier to maintain CI script.

Reply:
Yea, I added it in case we want to extend it later - ok, removed for now